### PR TITLE
Fix: Changes font color from gray->dark gray

### LINKED ISSUE #5951: Fonts: Change light gray font to dark gray across app
There are a number of places with light gray color. This is often difficult to see, especially if it is not bold. However, there is no point of making more and more text bold. 

Please change light gray color to a dark gray that is easier to read.

### PR DESCRIPTION
**Pull request by nlok5923**
Fixes #5951 

#### Short description of what this resolves:
This pr changes the color of font from gray to light gray.

#### Changes proposed in this pull request:

- font color changes

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
The change should be in one place and not more than 5 lines should be changed

File: app/styles/libs/stripe_connect.scss [hunk 1]
```diff
@@ -36,7 +36,7 @@
   background-image: linear-gradient(#7dc5ee, #008cdd 85%, #30a2e4);
   border-radius: 3px;
   box-shadow: inset 0 1px 0 rgba(255, 255, 255, .25);
-  color: #fff;
+  color:	#fff;
```
Comment (maze-runnar) [inline app/styles/libs/stripe_connect.scss:39]: revert unwanted changes

**Comment by nlok5923** [comment]
@iamareebjamal  please review this pull request and let me know of any changes needed.

**Comment by iamareebjamal** [comment]
Too many changes

**Comment by iamareebjamal** [comment]
Completed in other places

**Pull request closed without merging**